src/code-block.js

```diff
@@ -15,7 +15,10 @@ const preload = (content, resourcePath) => {
 
   codeBlocks.each((idx, elem) => {
     const $elem = $(elem);
-    const rawContent = $elem.html().toString();
+    let rawContent = $elem.html().toString();
+    if (shared.codeBlockLanguagesWithGenerics.includes($elem.attr('languageType'))) {
+      rawContent = rawContent.replace(/<\/(.*)>/, '');
```

Comment:
this regex will match the followigin `</some tag> adlskgnaut 1 > 2` and will include that other closing tag.  I think you need something like `/<\/(.*?)>/g` to match all "closing" tags?